packages/ed25519-signature-2018/package.json

```diff
@@ -44,8 +44,9 @@
   },
   "dependencies": {
     "@transmute/ed25519-key-pair": "0.7.0-unstable.2",
-    "@transmute/jose-ld": "^0.7.0-unstable.42",
-    "@transmute/security-context": "^0.7.0-unstable.42",
+    "@transmute/jose-ld": "^0.7.0-unstable.41",
```

Comment:
hmm you are downgrading versions here... this will prevent lerna bootstrap from working properly.. these should all be latests.

Reply:
Sorry didn't mean to do that! Will get it upgrade to latest!